prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -1,16 +1,30 @@
 presubmits:
   GoogleCloudPlatform/oss-test-infra:
-  - name: pull-prow-config-validate
-    branches:
-    - master
-    decorate: true
-    always_run: true
-    spec:
-      containers:
-      - image: gcr.io/k8s-prow/checkconfig:latest
-        imagePullPolicy: Always
-        command:
-        - /checkconfig
-        args:
-        - --config-path=./prow/config.yaml
-        - --job-config-path=./prow/prowjobs
+
+    - name: pull-prow-config-validate
+      branches:
+        - master
+      decorate: true
+      always_run: true
+      spec:
+        containers:
+          - image: gcr.io/k8s-prow/checkconfig:latest
```

Comment:
You likely want to make this version match the deployed version of Prow that this config is used in, to make sure that you get valid checks.

Reply:
👍 . I was going to add it in a separate PR, but it is small enough to include here.